docs/resources/Guild_Scheduled_Event.md

```diff
@@ -0,0 +1,126 @@
+# Guild Scheduled Event
+
+A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
+
+### Guild Scheduled Event Object
+
+###### Guild Scheduled Event Structure
+
+| Field                | Type                                                                                                                           | Description                                                                                         |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| id                   | snowflake                                                                                                                      | the id of the scheduled event                                                                       |
```

Comment:
```suggestion
| id                   | snowflake                                                                                                                      | the ID of the scheduled event                                                                       |
```

Reply:
all of the existing id descriptions use lowercase "id"

Reply:
all of the existing id descriptions use lowercase "id"

Reply:
> all of the existing id descriptions use lowercase "id"

well...
![image](https://user-images.githubusercontent.com/79790819/141755040-f0d14c00-26cc-42ea-b498-43770edc6d6f.png)

Reply:
that table might be an exception, but its typically lowercase (see guild, channel, emoji, invite, etc)

Reply:
is there any problem if i open a pr to change id to uppercase?

Reply:
Seems fine for consistency reasons

Reply:
i'll open as soon as i can

Reply:
I'm not sure about this..
Let discord decide it